### PR TITLE
Removed weave url

### PR DESCRIPTION
**Pull request by Darshnadas**
Removed the weave URL and added a note. In accordance with issue #31168

### CODE REVIEW

File: content/en/docs/setup/production-environment/tools/kubeadm/high-availability.md [hunk 1]
```diff
@@ -167,14 +167,13 @@ option. Your cluster requirements may need a different configuration.
...
-    In this example we are using Weave Net:
-
-    ```sh
-    kubectl apply -f "https://cloud.weave.works/k8s/net?k8s-version=$(kubectl version | base64 | tr -d '\n')"
-    ```
+  
+  {{<note>}}
+  You must pick your own CNI plugin and deploy it before you move on to next step.
+  If you don't do this, you will not be able to launch your cluster properly.
+  {{<note>}}
```
Comment (neolit123) [inline content/en/docs/setup/production-environment/tools/kubeadm/high-availability.md:176]: this note might not be needed, but deferring to the SIG docs reviewers to see what they think.
the note is indeed true, but in `Apply the CNI plugin of your choice:` we already link to a step that is implicitly required.

**Review by neolit123 — COMMENTED** [review]
thanks for the PR.

/sig cluster-lifecycle
/kind cleanup

**Comment by neolit123** [comment]
> Removed the weave URL and added a note. In accordance with issue #31168

if you add `fixes #31168` on a new line in the description it will auto-close the issue when this PR merges.

File: content/en/docs/setup/production-environment/tools/kubeadm/high-availability.md [hunk 2]
```diff
@@ -167,14 +167,13 @@ option. Your cluster requirements may need a different configuration.
     {{< /caution >}}
 
 1.  Apply the CNI plugin of your choice:
-    [Follow these instructions](/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm/#pod-network)
+    [Follow these instructions](/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm/#pod-network)
```
[the remark this answers is not in the document]
Comment (Darshnadas) [reply content/en/docs/setup/production-environment/tools/kubeadm/high-availability.md:170]: sure

File: content/en/docs/setup/production-environment/tools/kubeadm/high-availability.md (hunk 1 above)
Comment (Darshnadas) [reply content/en/docs/setup/production-environment/tools/kubeadm/high-availability.md:176]: So, what shall I note in there?

**Comment by Darshnadas** [comment]
> > Removed the weave URL and added a note. In accordance with issue #31168
> 
> if you add `fixes #31168` on a new line in the description it will auto-close the issue when this PR merges.

okay, then I'll add it like that

File: content/en/docs/setup/production-environment/tools/kubeadm/high-availability.md (hunk 1 above)
Comment (neolit123) [reply content/en/docs/setup/production-environment/tools/kubeadm/high-availability.md:176]: potentially, no extra note is needed.

File: content/en/docs/setup/production-environment/tools/kubeadm/high-availability.md [hunk 3]
```diff
@@ -167,14 +167,13 @@ option. Your cluster requirements may need a different configuration.
...
     to install the CNI provider. Make sure the configuration corresponds to the Pod CIDR specified in the kubeadm configuration file if applicable.
-
-    In this example we are using Weave Net:
-
-    ```sh
-    kubectl apply -f "https://cloud.weave.works/k8s/net?k8s-version=$(kubectl version | base64 | tr -d '\n')"
-    ```
+  
+  {{<note>}}
+  You must pick your own CNI plugin and deploy it before you move on to next step.
```
Comment (sftim) [inline content/en/docs/setup/production-environment/tools/kubeadm/high-availability.md:174]: I'd write “you must pick a network plugin that suits your use case” or something like that. Readers might think that they should somehow actually own a network plugin.

File: content/en/docs/setup/production-environment/tools/kubeadm/high-availability.md (hunk 1 above)
Comment (sftim) [reply content/en/docs/setup/production-environment/tools/kubeadm/high-availability.md:176]: The indentation of this note doesn't look right to me.

I would leave the note in - people do overlook this detail.

**Review by sftim — COMMENTED** [review]
My thoughts on this.

File: content/en/docs/setup/production-environment/tools/kubeadm/high-availability.md (hunk 1 above)
Comment (sftim) [reply content/en/docs/setup/production-environment/tools/kubeadm/high-availability.md:176]: Try indenting the note to 4 spaces. It should match the indentation of other text in the list item.

**Comment by Darshnadas** [comment]
okay @neolit123

File: content/en/docs/setup/production-environment/tools/kubeadm/high-availability.md (hunk 3 above)
Comment (Darshnadas) [reply content/en/docs/setup/production-environment/tools/kubeadm/high-availability.md:174]: okay @sftim

File: content/en/docs/setup/production-environment/tools/kubeadm/high-availability.md (hunk 1 above)
Comment (Darshnadas) [reply content/en/docs/setup/production-environment/tools/kubeadm/high-availability.md:176]: okay @sftim

**Review by neolit123 — COMMENTED** [review]
LGTM, thanks

File: content/en/docs/setup/production-environment/tools/kubeadm/high-availability.md [hunk 4]
```diff
@@ -167,14 +167,13 @@ option. Your cluster requirements may need a different configuration.
...
-    In this example we are using Weave Net:
-
-    ```sh
-    kubectl apply -f "https://cloud.weave.works/k8s/net?k8s-version=$(kubectl version | base64 | tr -d '\n')"
-    ```
+  
+  {{< note >}}
+  You must pick a network plugin that suits your use case and deploy it before you move on to next step.
+  If you don't do this, you will not be able to launch your cluster properly.
+  {{< /note >}}
```
Comment (sftim) [inline content/en/docs/setup/production-environment/tools/kubeadm/high-availability.md:176]: I'd expect to see:
```suggestion
   {{< note >}}
   You must pick a network plugin that suits your use case and deploy it before you move on to next step.
   If you don't do this, you will not be able to launch your cluster properly.
   {{< /note >}}
```

Comment (Darshnadas) [reply content/en/docs/setup/production-environment/tools/kubeadm/high-availability.md:176]: okay @sftim

File: content/en/docs/setup/production-environment/tools/kubeadm/high-availability.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (sftim) [reply content/en/docs/setup/production-environment/tools/kubeadm/high-availability.md:170]: We can live with it.

**Review by sftim — COMMENTED** [review]
/approve

**Comment by sftim** [comment]
/lgtm
also from https://github.com/kubernetes/website/pull/31325#pullrequestreview-853983965

**Pull request merged**